src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/servicebus/domains/payments/PaymentsPublisher.java

```diff
@@ -59,4 +60,17 @@ public void send(PaymentCommand cmd) {
             );
         }
     }
+
+    private void doSend(IMessage message, boolean retry) throws ServiceBusException, InterruptedException {
+        try {
+            queueClient.send(message);
+        } catch (Exception ex) {
+            if (retry) {
+                doSend(message, false);
+            } else {
+                throw ex;
+            }
+
+        }
+    }
```

Comment:
from library i see specific mentioning on retry and suppose to use special method:

```java
} catch (ServiceBusException exception) {
    if (exception.getIsTransient() && retry) {
        doSend;
    }
}
```

Reply:
first I thought we should retry for transient exception as library  but I changed my mind as we reduced the timeout  we can try for all exceptions.

Reply:
i see. perhaps worth doing at most 10s as callback is for 30s. all operations with idam and ccd will consume most of that time. hopefully retry will fit in

Reply:
I suggest not to catch any exception from the queue client but investigate how retry policy works and if necessary apply a customized policy.

Reply:
I believe if it works it should work less than a second but I am not sure about the side effects.
we can make operation timeout 10s.

Reply:
As Andrey said, retry policy would be a nice thing to try first. It would be good if that policy was applied to all publishing, including `processed-envelopes` queue.

Reply:
queueclient has default retry policy already running.

ConnectionStringBuilder

`public RetryPolicy getRetryPolicy()
	{
		return (this.retryPolicy == null ? RetryPolicy.getDefault() : this.retryPolicy);
	}`

Reply:
can we just adjust the timeout  as a first before retrying it manually ?

Reply:
and see how the system behaves in prod.

Reply:
increase or decrease ?

Reply:
It is necessary to work out how the default retry policy (which is in place and applies some kind of exponentially increasing delays) handles the timeout value. Having these prod failures means that this policy or its params are not suitable for all prod situations. It is worth looking through their code and even running some tests (with mocks) to ensure its behaviour and then change params or the policy itself (some change is definitely necessary).